File: Array With Elements Not Equal to Average of Neighbors..js
var rearrangeArray = function(nums) {
 nums.sort((a, b) => a-b); 
  let result = [];
  let i = 0;
  let j= nums.length-1;  
  while(i < j){
    result.push(nums[i++]);
    result.push(nums[j--]);
  }  
  if(i == j) result.push(nums[i]);   
  return result;
};

File: 2085-array-with-elements-not-equal-to-average-of-neighbors/array-with-elements-not-equal-to-average-of-neighbors.py
class Solution(object):
    def rearrangeArray(self, nums):
        nums.sort()
        result = []
        i, j = 0, len(nums) - 1
        
        while i < j:
            result.append(nums[i])
            result.append(nums[j])
            i += 1
            j -= 1
        
        if i == j:
            result.append(nums[i])
        
        return result
        